app/src/ui/lib/filter-list.tsx

```diff
@@ -340,7 +341,14 @@ export class FilterList<T extends IFilterListItem> extends React.Component<
       return
     }
 
-    if (event.key === 'ArrowDown') {
+    const filterText = this.props.filterText
+    if (key === 'Escape' && (filterText && filterText.length !== 0)) {
```

Comment:
This section would appear to be redundant now since it's handled in the TextBox. Could we remove it please?